pkg/provider/isolation.go

```diff
@@ -36,13 +36,13 @@ func AreResourcesIdentical(p *Pod) bool {
 		memoryLimits := cut.Resources.Limits.Memory()
 
 		// Check for mismatches
-		if cpuRequests.Value() != cpuLimits.Value() {
-			logrus.Debugf("%s has CPU requests %d and limits %d that do not match.", cut.String(), cpuRequests.Value(), cpuLimits.Value())
+		if cpuRequests.AsApproximateFloat64() != cpuLimits.AsApproximateFloat64() {
```

Comment:
I'd avoid the use of float here. It looks a bit strange to have to perform this casting just for comparing two quantities. You can use the `Equal()` function of the `resource` package.

Reply:
I will change it.  Thanks!